experiments/test_horizon.py

```diff
@@ -209,10 +209,6 @@ def show_horizon(rob, height=64, horizon_dist=200):
 def main():
     mc, rob = start_mission()
     mc.safeStart()
-    MalmoPython.setLogging("log.txt", MalmoPython.LoggingSeverityLevel.LOG_FINE)
-    MalmoPython.setLoggingComponent(MalmoPython.LoggingComponent.LOG_VIDEO, True)
-    MalmoPython.setLoggingComponent(MalmoPython.LoggingComponent.LOG_TCP, True)
-    MalmoPython.setLoggingComponent(MalmoPython.LoggingComponent.LOG_AGENTHOST, True)
```

Comment:
Is there an alternative to MalmoPython logging?

Reply:
There is no such interface in python version of the library. It has logging mechanism throughtout standart logger, but I don't know if it implements features similar to MalmoPython.

Reply:
I've added logger from `__init__.py` to test_horizon.py